doc/developers/contributing.rst

```diff
@@ -34,6 +34,14 @@ See :ref:`new_contributors` to get started.
 
 |
 
+
+In case you experience issues using this package, do not hesitate to submit a
+ticket to the
+`Bug Tracker <https://github.com/scikit-learn/scikit-learn/issues>`_. You are
```

Comment:
"Github Issue tracker" (just to make it explicit that issues are tracked on Github without having to inspect the URL)